package.json

```diff
@@ -21,7 +21,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react-hooks": "^5.0.0",
-    "patch-package": "8.0.0",
+    "patch-package": "^8.0.0",
```

Comment:
**question:** why this change?

Reply:
It happened automatically when I installed `patch-package`. I didn't notice that it was already installed here. I can revert that if you wish